Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -1,28 +1,42 @@
 """This contains the plotting and geo data management methods for the COVIDcast signals."""
 
 import os
-from datetime import date
 
 import geopandas as gpd
 import pandas as pd
 
+from covidcast import covidcast
```

Comment:
Can/should this import be reworded using .-notation to match `__init__.py`?

Reply:
I don't have a strong preference, if convention is to use relative imports we should stick with that for consistency.